app/api/alibaba.ts

```
@@ -79,32 +78,83 @@ async function request(req: NextRequest) {
+
+          // Remove input.messages to avoid duplication
+          delete jsonBody.input;
+
+          jsonBody.stream = true;
+        }
+
+        const current_model = jsonBody?.model;
+        console.log("[Alibaba] custom models", current_model);
+
+        //kiem tra xem model co phai la qwen-vl hay khong (vision model)
+        if (current_model && current_model.startsWith("qwen-vl")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)